server/bin/unittests

```diff
@@ -315,6 +323,7 @@ function _setup_state {
     # are found in $_testopt/unittest-scripts.
     _orig_PATH=$PATH
     export PATH=$_testopt/unittest-scripts:$_testopt/bin:$PATH
+    export PYTHONPATH=$_testopt/unittest-lib:$PYTHONPATH
```

Comment:
You copy into separate directories for "server" and "satellite", but there's a single `PYTHONPATH` modification. Is that really what we want? Is there any need for separate Python overlay libraries? Is doesn't appear there is right now, so maybe reserve that for later and just make the single copy you're using?

Reply:
At the risk of belaboring this patch, even though it does still need a rebase, this seems like a good catch that we should address.

We handle the `PATH` for the satellite script invocation context separately at: https://github.com/distributed-system-analysis/pbench/blob/3219eea0d1fce179cd1d71ef3b6a1227ec57c842/server/bin/unittests#L111

So we should similarly setup `PYTHONPATH` to add the satellite test libs there as well.